src/test/java/DZ_13DBTest.java

```diff
@@ -0,0 +1,68 @@
+import org.example.automation.Config;
+import org.example.automation.Session;
+import org.example.automation.base.BaseTestNG;
+import org.testng.annotations.Test;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+public class DZ_13DBTest extends BaseTestNG {
+    // UPDATE, DELETE
+
+    @Test(testName = "DataBaseSELECTTest")
+    public void testSelect() {
+        try {
+            PreparedStatement statement = Session.get().mysql().preparedStatement("SELECT * FROM homework_user_data WHERE id=?;");
+            statement.setInt(1, 1);
+            ResultSet resultSet = statement.executeQuery();
+            System.out.println(resultSet);
```

Comment:
So may be you need to add some assets here because you need to check that it does select proper record